Tests/TuistGeneratorTests/ProjectMappers/AutogeneratedSchemesProjectMapperTests.swift

```diff
@@ -30,6 +30,16 @@ final class AutogeneratedSchemesProjectMapperTests: TuistUnitTestCase {
             product: .unitTests,
```

Comment:
It's worth adding another test case to reproduce the scenario you described in the issue to verify the fix addresses it :)

e.g.

`ATests` > `[A, B]`

Reply:
I covered it with my first test case to make sure only the proper targets are picked up. Is that enough for you @kwridan ?